<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HookPeakingForm</class>
 <widget class="GUIFormBase" name="HookPeakingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>42</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="toolButtonMode">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="HookPeaking.qrc">
     <normaloff>:/Resources/Resources/PeakingN.png</normaloff>
     <normalon>:/Resources/Resources/PeakingS.png</normalon>:/Resources/Resources/PeakingN.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>42</width>
     <height>42</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonColor">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderTransparentRate">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="pageStep">
    <number>8</number>
   </property>
   <property name="value">
    <number>80</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelTransparent">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="HookPeaking.qrc"/>
 </resources>
 <connections/>
</ui>
